projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -571,4 +572,8 @@ export class AppExtensionService implements RuleContext {
 
     return true;
   }
+
+  resetBulkActions(): void {
```

Comment:
Do we want it this to be exposed to extension service? I think selected action can be reset upon executing an action in dropdown component

Reply:
@MichalKinas The Bulk Actions Dropdown is visible above the dialogs and resetting it after executing an action from the component would make the selection not visible at all. That's the only reason for using the Extension Service, because it allows resetting selection on dialog's `afterClosed`. Is it okay in this case or should I look for another solution, possibly through another actions executed from ADW level?

Reply:
Okay if the design requires it then it is fine, I would maybe rename it a little bit since now it's not clear why extension service would want to reset bulk actions and it creates a tight-coupling with the bulk dropdown component so instead let's rename it to `bulkActionExecuted` subject which will be a generic subject not coupled directly to any component and then the dropdown component can just reset whenever this event will happen, later on this subject can be even reused by different components if they will need to react on bulk action being executed, does this approach sound good to you?